ui/src/Grid/Message.elm

```diff
@@ -0,0 +1,7 @@
+module Grid.Message exposing (..)
+
+import Grid.Player exposing (..)
+
+type Msg = PlayerMoved PlayerId Position Orientation
+         | PlayerDied PlayerId
+         | NoOp
```

Comment:
client would have the same message types that of server .. right ?
`NoOp` is Msg type .. i see that people use it whenever they have unnecessary conditions..
as `view` or standard functions have to have to return `Html Msg`